_docs/11-training.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: splash
+title: "Training"
+permalink: /docs/training/
+hidden: true
+header:
+  overlay_color: "#5e616c"
+  overlay_image: /assets/images/201106-CORP-BNR-Design-Centers-FPGAs-and-plds-Banner-2880x280.jpg
+excerpt: "Welcome to the Training page."
```

Comment:
This "Welcome ..." excerpt on each of the sub-pages may need some attention from KK to provide a focused introduction to the content of the pages. Ciaran, could you note this as an issue to resolve with KK.

Reply:
Sure, will bring it up with KK offline.